### PR TITLE
0.1.4 Bulk Rework

### PR DESCRIPTION
**Pull request by LT-Rascek**
0.1.4 Map (and a few Mechanics) Rework

(Apparently, Github ate my old to do list, so I'll rewrite it again/update the changelog as well)

### CODE REVIEW

File: ChangeLog.txt [hunk 1]
```diff
@@ -1,11 +1,402 @@
...
+   - Turuk/Gokturk culture removed for 476
 
-BUGFIXES:
+Religion:
+- Doctrines:
+   - Right to Rule Doctrines: (Credit: Rajas of Asia)
+      - All faiths have a "Right to Rule" doctrine, which influences which type of government they may have
+      - If you have a religious doctrine OR cultural tradition that supports your government, you _cannot_ change it
+      - Otherwise, a decision is available to change your government
+      - Options: Divine Right (Feudal government), Material (Clan government), Rajadharma (Mandala government), Khvarenah (Eranshar government)
```
Comment (loup99) [inline ChangeLog.txt:153]:
```suggestion
      - Options: Divine Right (Feudal government), Material (Clan government), Rajadharma (Rajamandala government), Khvarenah (Eranshar government)
```

File: ChangeLog.txt [hunk 2]
```diff
@@ -1,11 +1,402 @@
...
+- Germania Magna:
+   - Adjusted borders of many tribal powers to better match historical records, include Saxony, Thuringia, Alamannia, Lombards, Venedi
+   - Added Suavi vassal in the eastern part of the Alamannian Confederation, per Jordanes
+   - Culture/Religion:
+      - Chatti moved to a more accurate region for Vth Century
+      - Cheruscii removed as they were likely no longer extant
+- Pontic-Caspian Steppe:
+   - Improved the position of the Bolghar tribes
+   - Acatiri are a Hunnic tributary to start
+   - Improved the division of Crimea between the Bosporan Kingdom, the (Crimean) Goths, and Byzantium
```
Comment (loup99) [inline ChangeLog.txt:140]:
```suggestion
   - Improved the division of Crimea between the Bosporan Kingdom, the (Crimean) Goths, and ERE
```

File: WtWSMS/common/casus_belli_types/T4N_punative_war.txt [hunk 3]
```diff
@@ -0,0 +1,239 @@
+﻿punative_war_cb = {
```
Comment (loup99) [inline WtWSMS/common/casus_belli_types/T4N_punative_war.txt:1]: Typo here and in the file title, no?

File: WtWSMS/common/culture/creation_names/BP_ck2_custom_names.txt [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+				CULTURE_TRIGGER = occitan_culture_trigger
+				CULTURE_REGION = custom_catalan
+				REGION_LOCKED = yes
+			}
+		}
+		scope:other_culture = culture:romano_hispanic
+	}
+	hybrid = yes
+}
+#WtWSMS_pots.62 - Portuguese (Not ported for now)
```
Comment (loup99) [inline WtWSMS/common/culture/creation_names/BP_ck2_custom_names.txt:95]: Any reason for why this culture in particular isn't ported? Not to add work but just to understand the underlying choice

Comment (LT-Rascek) [reply WtWSMS/common/culture/creation_names/BP_ck2_custom_names.txt:95]: Portuguese has their own Vanilla decision, like English culture.

See: https://ck3.paradoxwikis.com/Decisions
Cross-Reference "Found Portugal"

Comment (loup99) [reply WtWSMS/common/culture/creation_names/BP_ck2_custom_names.txt:95]: In my view relying on that doesn't make much sense given that the melting-pots are meant to cover the entire area, that decision corresponds to the 1066 setup, not the mod.

Comment (LT-Rascek) [reply WtWSMS/common/culture/creation_names/BP_ck2_custom_names.txt:95]: Except Portuguese itself is more of a divergence from the earlier Galician culture/language. That' doesn't quite match up with history or WtWSMS_pots.62; WtWSMS_pots.62 pretty much "instantly" creates these cultures from Romano-Hispanic, whereas I think we'd have Galician evolve from Romano-Hispanic and Portuguese branch from Galician.

File: WtWSMS/common/casus_belli_types/T4N_punative_war.txt (hunk 3 above)
Comment (LT-Rascek) [reply WtWSMS/common/casus_belli_types/T4N_punative_war.txt:1]: Will need to reinstall the spell-checker.

**Pull request closed without merging**